support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -194,38 +189,23 @@ export function SupporterPlusLandingPage({
 					reporting on the events shaping our world.
 				</p>
 			</CheckoutHeading>
-			<Container
-				sideBorders
-				cssOverrides={
-					optimisedMobileLayout
-						? darkBackgroundContainerMobile
-						: backgroundContainer
-				}
-			>
+			<Container sideBorders cssOverrides={darkBackgroundContainerMobile}>
```

Comment:
Is the `backgroundContainer` css being used elsewhere? Or can that be stripped out now?

Reply:
Yes have merged them into `darkBackgroundContainerMobile`. Thanks